### PR TITLE
approve,reject,delete user with api

### PR DESCRIPTION
**Pull request by fknraini**
Did you write good & comprehensive specs? ⭐️

What's this PR do ? ⭐️

What are the relevant tickets ? ⭐️

Where should the reviewer start ?

How should this be manually tested ?

What is the Dockup URL ?

Any background context you want to provide ?

Any deployment note (migrations, new envs, rake tasks, cron) ?

Screenshots (if appropriate)

Definition of Done:

[ ] Specs / Tests are adequate ? ⭐️

[ ] Changes conform with code quality tools (Rubocop, Reek, Brakeman) ? ⭐️

[ ] Does this PR require Database changes ?

[ ] Does this PR require monitoring after deployment (Scout, logs, Sentry, etc) ?

[ ] Does this PR require a regression test by QA ? All fixes require a regression test.

[ ] Does this add new dependencies ? (network setup, new 3rd party services)

### CODE REVIEW

File: src/Actions/adminAction.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  const config = {
+    headers: {
+      "Content-type": "application/json",
+      "Authorization": "Bearer 125|ALFUiv47Lm48lqRnzLHuPDjiNvo4qV6d47VbEiuf",
+    },
+  };
+
+  return (dispatch) => {
+    axios
+      .get("https://floating-journey-97236.herokuapp.com/api/v1/users", config)
```
Comment (raendyelvicar) [inline src/Actions/adminAction.js:18]: For url API can use process.env.REACT_APP_API_URL + "/path" (can look at user.service.js file for references)

File: src/Actions/adminAction.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import axios from "axios";
+
+export const GET_USERS_LIST = "GET_USERS_LIST";
+
+export const getUsersList = () => {
+
+  const config = {
+    headers: {
+      "Content-type": "application/json",
+      "Authorization": "Bearer 125|ALFUiv47Lm48lqRnzLHuPDjiNvo4qV6d47VbEiuf",
```
Comment (raendyelvicar) [inline src/Actions/adminAction.js:12]: Authorization can import authheader() function from Helpers folder (auth-header.js file), for references can look at user.service.js

File: src/Actions/adminAction.js (hunk 1 above)
Comment (raendyelvicar) [reply src/Actions/adminAction.js:18]: add REACT_APP_API_URL variable at .env file

File: src/Styles/Login.css [hunk 3]
```diff
@@ -122,5 +122,5 @@
     padding: 1rem;
     width: 60%;
   }
-}
+} 
 /* # sourceMappingURL=Login.css.map */
```
Comment (herulhidayat) [inline src/Styles/Login.css:126]: please remove unused code

File: src/Styles/Table.css [hunk 4]
```diff
@@ -1,5 +1,5 @@
 .container {
-    background-color: #fff;
+    /* background-color: #fff; */
```
Comment (herulhidayat) [inline src/Styles/Table.css:2]: please remove unused code

File: src/Actions/adminAction.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+      });
+  };
+}
+
+export const rejectUser = (uid) => {
+  const config = {
+    headers: {
+      "Content-type": "application/json",
+      "Accept" : "application/json",
+      "Authorization": "Bearer 125|ALFUiv47Lm48lqRnzLHuPDjiNvo4qV6d47VbEiuf",
```
Comment (chaterinamf) [inline src/Actions/adminAction.js:91]: Doesn't the value of the token differ every time you log in?

**Review by herulhidayat — APPROVED** [review]

**Review by andranugraha — APPROVED** [review]

File: src/Actions/adminAction.js (hunk 5 above)
Comment (fknraini) [reply src/Actions/adminAction.js]: solved

**Pull request closed without merging**